assets/static/js/common/dise_api.js

```diff
@@ -1,6 +1,6 @@
 (function() {
-    var base = 'https://dise.klp.org.in/api/';
-    var DEFAULT_ACADEMIC_YEAR = '13-14';
+    var base = 'http://dise.olp.ind.in/api/';
+    var DEFAULT_ACADEMIC_YEAR = '15-16';
```

Comment:
Please remember to either change this into a function which calculates the APIs dynamically based on a `State` selection or to just change this to point wherever you're hosting your `dise_dashboard` instance.